src/seedu/addressbook/data/person/Person.java

```diff
@@ -54,6 +59,8 @@ public Address getAddress() {
         return address;
     }
 
+    public int getSequenceNumber() { return sequenceNumber; }
```

Comment:
Please check our coding standards on how to write methods that contain a single line of code.